ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,303 @@
+                                    </desc>
+                                </item>
+                                <item xml:id="a3">
+                                    <locus target="#5r"/>
+                                    <desc type="Inventory"/>
+                                    <q xml:lang="fr">n: 88 Antoine d'Abbadie </q>
+                                </item>
+                                <item xml:id="a4">
+                                    <locus target="#5r"/>
+                                    <desc type="PurchaseNote"/>
+                                    <q xml:lang="gez">ዝ፡ መጽሐፍ᎓ ዘእንጦንዮስ፡ ወልደ፡ አባዲ፡ ፍራንሳዊ᎓ ዘተሣየተ፡ እምኀበ፡ አባ፡ ወልደ፡ ጊዮርጊስ፡
+                                    </q>
+                                    <q xml:lang="en">This book, bought from Father <persName ref="PRS13644WaldaGiyorgis"/>,  belongs to <persName  ref="PRS1071dAbbadi" role="owner"/>, the Frenchman.</q>
+                                </item>
+                                <item xml:id="a5">
+                                    <locus target="#5ra"/> 
+                                    <desc type="GuestText">
+                                        <q xml:lang="gez">ወይሄሉ፡ እስከ፡ ለዓለመ፡ ዓለም፡</q>
+                                    </desc>
+                                </item>
+                                <item xml:id="a6">
+                                    <locus target="#5ra"/> 
+                                    <desc type="Unclear">
+                                        <q xml:lang="gez">ፈንሐስ፡ ሚናስ፡ ናዖድ፡ ሙዙ፡ ፆታ፡ ማርያም፡ ውቢት፡ ሰብለ፡ ማርያም፡ ሚካኤል፡ ጽጌ፡ አረጋዊ፡ በከፋ፡</q>
+                                    </desc>
+                                    <note> Depending on the available space, these names are added on fols. 9v, 51r and 71r.</note>
+                                </item>
+                                <item xml:id="a7">
+                                    <locus target="#5ra"/> 
+                                    <desc type="Supplication">
+                                        <q xml:lang="gez">ተማሕፀንኩ፡ በእምነ፡ ወለተ፡ ጴጥሮስ፡ ኃጥዕ፡ ወልደ፡ ጊዮርጊስ፡ ነፍስየ፡ እምነፍስኪ፡ ኢትሌልያ፡  ለዓለመ፡ ዓለም፡ አሜን፡ ለይኩን፡ ለይኩን። የተጻፈበት፡ ወቄት፡ ታ፭፡ ጨው፡</q>
+                                        <q xml:lang="eng">I, the sinner <persName ref="PRS13644WaldaGiyorgis" role="owner"/>, entrust myself to our mother <persName ref="PRS9848Walatta"/>, 'Do not separate my soul from yours in eternity, amen!' May it be thus, may it be thus!</q>
+                                    </desc>
+                                </item>
+                                <item xml:id="a8">
+                                    <locus target="#75ra"/>
+                                    <desc type="Correction"> The text of <ref target="#a6"/> has been erased, with the exception of the first word. Instead of the erased names was written <persName ref="PRS1071dAbbadi" role="owner"> እንጦንዮስ.</persName>.</desc>
```

Comment:
In this case, since there is no fixed label, it is better to write the text of the link as `<ref target="#a6">Additio 6</a`>